content/guides/core/automations/automation-events.md

```diff
@@ -86,6 +84,14 @@ For **Run metrics change threshold met** events, you can configure:
 
 The current and prior window do not overlap. By default, the current window includes the 10 most recent logged values, and the prior window includes the 50 logged values prior to the current window.
 
+#### Run filters
+This section describes how the automation selects runs to evaluate.
+
+- Each run is considered separately, and the automation can potentially start due to the result of multiple runs.
+- Each run's values are put into a separate window and compared to the threshold separately.
+- If you specify a run filter, only matching runs can start the automation.
+- A particular run metric automation runs at maximum once per run per 24-hour period.
```

Comment:
I think this is confusing because `run` is ambiguous, and eg could imply "a particular run metric can only evaluate metric values once per day".  We use the terms "fire" or "trigger" to refer to what happens when we execute the action that the user  has specified to avoid the confusion.

Reply:
PTAL to see if this is better.